### PR TITLE
Bug - Footer

### PR DESCRIPTION
**Pull request by alexlewis9**
https://linear.app/uoft-blueprint-internal/issue/INTL-35/bug-footer

create footer that works on mobile view and remove the extra white space

### CODE REVIEW

**Review by jordanjanakievski — COMMENTED** [review]
Looks good. Text is a little small are the larger screens but I don't hate it. Maybe another opinion would be good.
Screenshot 2024-11-20 at 17 22 05

**Comment by alexlewis9** [comment]
I've changed the text sizing a bit but I don't have a screen that wide so please let me know if it looks better.

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,41 +1,46 @@
...
+      className="footer-link"
+    >
+      {link.label}
+    </Anchor>
+  ));
+
   return (
-    <footer className="main-footer">
-      <Group className="footer-menu">
+    <div className="main-footer">
```
Comment (jordanjanakievski) [inline src/components/Footer/Footer.tsx]: Lets keep this div called footer. Its an accessibility thing and good to have.

It works the same as div

**Review by jordanjanakievski — COMMENTED** [review]
Final thing. Missed this the first time. Font is looking good to me right now

**Review by jordanjanakievski — APPROVED** [review]

**Pull request merged**